src/ppx/xr_component.cpp

```diff
@@ -727,36 +727,46 @@ void XrComponent::EndFrame(const std::vector<grfx::SwapchainPtr>& swapchains, ui
         layers.data(),                        // layers
     };
 
-    CHECK_XR_CALL(xrEndFrame(mSession, &frameEndInfo));
+    auto a = xrEndFrame(mSession, &frameEndInfo);
```

Comment:
can you give it a descriptive name? like `res` or `ret` or similar? Or revert to the previous behaviour